resources/definitions/ultimaker_s6.def.json

```diff
@@ -0,0 +1,53 @@
+{
+    "version": 2,
+    "name": "UltiMaker S6",
+    "inherits": "ultimaker_s8",
+    "metadata":
+    {
+        "visible": true,
+        "author": "UltiMaker",
+        "manufacturer": "Ultimaker B.V.",
+        "file_formats": "application/x-ufp;text/x-gcode",
+        "platform": "ultimaker_s7_platform.obj",
```

Comment:
Is it really the exact same buildplate ?

Reply:
You are correct, should have been the S5 platform. I will adjust. I also noticed that the backpanel is not showing correctly and needs to be adjusted.